crate/diffs/type_cosplay.diff

```diff
@@ -16,14 +16,25 @@ diff -r ./insecure/src/lib.rs ../../../../lints/type_cosplay/ui/insecure/src/lib
 Only in ../../../../lints/type_cosplay/ui/insecure/src: lib.stderr
 Only in ../../../../lints/type_cosplay/ui: insecure-2
 Only in ../../../../lints/type_cosplay/ui: insecure-3
+Only in ../../../../lints/type_cosplay/ui: insecure-anchor
+diff -r ./recommended/Cargo.toml ../../../../lints/type_cosplay/ui/recommended/Cargo.toml
+19c19
+< anchor-lang = "0.20.1"
+---
+> anchor-lang = "0.25.0"
 diff -r ./recommended/src/lib.rs ../../../../lints/type_cosplay/ui/recommended/src/lib.rs
-10c10,12
+10c10,17
 <     pub fn update_user(ctx: Context<UpdateUser>) -> ProgramResult {
 ---
 >     pub fn update_user(
 >         ctx: Context<UpdateUser>,
 >     ) -> anchor_lang::solana_program::entrypoint::ProgramResult {
```

Comment:
Let's have a `recommended-2` instead of modifying the existing `recommended`.

Reply:
So still keep the recommended example and just not test it?

Reply:
Sorry, I'm not following. Not test it why?

Reply:
I made the requested changes. I am testing it now. The reason I said not is because it will pass, but for the wrong reasons. That's because the recommended example in sealevel just annotates structs with #[account], they don't do any deserializations. So it's secure, but doesn't really represent a real world example. Nevertheless, still testing it and passing, but sorta for the wrong reasons.